public/js/containers/App.js

```diff
@@ -1,17 +1,18 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Actions } from 'jumpstate';
+import { bindActionCreators } from 'redux';
+import actions from 'actions';
 import Page from 'components/Page';
 import Filters from 'components/Filters/Filters';
 import Charts from 'components/Charts/Charts';
 
 class App extends Component {
     componentDidMount() {
-        Actions.fetchCommissioningDesksRedux();
+        this.props.actions.fetchCommissioningDesks();
     }
 
```

Comment:
Have the update function live here. Then the data flow is all pivoted around this point.